document/CoreFormat.mdk

```diff
@@ -348,38 +352,38 @@ This shall be one of the following:
 *	`Tiger`
 *	`Whirlpool`
 
-## `issue` object { #issue }
+## `result` object { #result }
 
-### General { #issue-General }
+### General { #result-General }
 
-An `issue` object describes a single issue detected by a static analysis tool.
+A `result` object describes a single result produced by a static analysis tool.
```

Comment:
As we discussed: This sentence says that the programmatic construct `result` represents the English-language concept of a "result", which is defined elsewhere (in the "Terms and Definitions" section).